libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -241,6 +246,9 @@ AP_OSD_Screen::AP_OSD_Screen()
 #define SYM_XERR      0xEE
 #define SYM_KN        0xF0
 #define SYM_NM        0xF1
+#define SYM_COL       0x3A
+#define SYM_FLY       0x9C
+#define SYM_ZERO      0x30
```

Comment:
0 is not enouth?

Reply:
sorry..tried but not skilled enough to properly put "0" in the write call

Reply:
figured it out...duh..removed symbol defs